client/src/app/modules/welcome/welcome.component.scss

```diff
@@ -0,0 +1,79 @@
+$orange-color: #F5612C;
+$blue-color: #2DADB7;
+.welcome-body {
+    height: 100vh;
+    width: 100%;
+    background-image: url(../../../assets/images/background/welcome-background.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.website-title {
+    font-family: cursive;
+    font-size: 3vw;
+    font-weight: bold;
+    color: $orange-color;
+    width: fit-content;
+    position: fixed;
+    z-index: 2;
+    margin-top: 3%;
+    margin-left: 8%;
+}
+
+.motto {
+    font-family: cursive;
+    font-size: 2vw;
+    color: $orange-color;
+    width: fit-content;
```

Comment:
Im not sure fit-content works in all browser, search about it please

Reply:
StackOverflow: 
width: fit-content is still in experimental stages, and is currently supported on Chrome 46.x and above (without vendor prefix), FF 3.x and above (with vendor prefix). Not supported on either IE or edge.
And what I did is to delete it because it is not necessary to have it in our code.